### PR TITLE
Include functions to get size and free space of arbitrary mounting points

### PR DESCRIPTION
**Pull request by marciopamplona**

### CODE REVIEW

File: src/mgos_vfs.c [hunk 1]
```diff
@@ -1002,3 +1002,25 @@ bool mgos_vfs_gc(const char *path) {
   LOG(LL_INFO, ("%s %p %d", path, fs, ret));
   return ret;
 }
+
+size_t mgos_get_fs_size_path(const char *path) {
```
Comment (rojer) [inline src/mgos_vfs.c:1006]: please rename to `mgos_vfs_get_space_total` and make `mgos_get_fs_size` use it

File: src/mgos_vfs.c [hunk 2]
```diff
@@ -1002,3 +1002,25 @@ bool mgos_vfs_gc(const char *path) {
...
+  struct mgos_vfs_mount_entry *me = find_mount_by_path("/", NULL);
+  if (me == NULL) return 0;
+  mgos_vfs_lock();
+  res = me->fs->ops->get_space_total(me->fs);
+  me->fs->refs--;
+  mgos_vfs_unlock();
+  return res;
+}
+
+size_t mgos_get_free_fs_size_path(const char *path) {
```
Comment (rojer) [inline src/mgos_vfs.c:1017]: ditto - rename to `mgos_vfs_get_space_free` and make `mgos_get_free_fs_size` use it

**Comment by rojer** [comment]
thank you, this makes sense, just two minor comments.

**Review by rojer — CHANGES_REQUESTED** [review]

**Comment by rojer** [comment]
closing in favor of https://github.com/mongoose-os-libs/vfs-common/pull/1

File: src/mgos_vfs.c (hunk 1 above)
Comment (fovea1959) [reply src/mgos_vfs.c:1006]: done.

File: src/mgos_vfs.c (hunk 2 above)
Comment (fovea1959) [reply src/mgos_vfs.c:1017]: done.

**Pull request closed without merging**